src/clojush/pushgp/genetic_operators.clj

```diff
@@ -350,7 +350,7 @@ be set globally or eliminated in the future."
         variant (diversifying? child-genome argmap)
         new-genome (if variant
                      child-genome
-                     (random-plush-genome max-genome-size-in-initial-program atom-generators argmap))]
+                     (random-plush-genome max-genome-size-in-initial-program atom-generators true argmap))]
```

Comment:
It seems awkward that we introduce an unnamed `true` here. Everything else here has a name that at least sort of suggests what it's doing, but that `true` just floats there without any connection to the world in a way that makes it hard to understand. Is there a (reasonable) way we could have this have a name here?